config/src/config/state_sync_config.rs

```diff
@@ -265,6 +267,7 @@ pub struct AptosDataClientConfig {
 impl Default for AptosDataClientConfig {
     fn default() -> Self {
         Self {
+            latency_monitor_loop_interval_ms: 50, // 50 milliseconds
```

Comment:
Is 50ms a reasonable number?

Reply:
Good question. When I look at the data, I see latency variance across 100ms buckets (which makes me reluctant to run this loop less frequently, because it'll skew the results, e.g., if I run it only every 200ms, then we won't get data on fine-grained improvements). Likewise, I don't want to run it too frequently, because of resource costs.

Do you have a specific concern? One thing we could do is make this less frequent in mainnet (if we're worried about resource usage?)

Reply:
Can this be overridden in config?

Reply:
Yeah, it can be. I can also add it to the config optimizer (so we have different defaults). But, I'm not sure there's much resource usage here at 50ms. Will wait for @bchocho to clarify the exact concern 😄